e2e/tests/omnibus-sepolia.toml/BFP_Market_ETH.e2e.js

```diff
@@ -420,4 +420,113 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
     log({ newCollateral });
     assert.equal(newCollateral.toNumber(), 0);
   });
+
+  describe('Liquidation', async () => {
```

Comment:
Ok I have just realised this is a nested describe. Please extract liquidation tests into their own separate files and copy necessary "bootstrapping" `it` steps to it.

Reply:
Made a new separate file for liquidation tests.